processor/src/stack/inputs.rs

```diff
@@ -0,0 +1,66 @@
+use super::{vec, Felt, Vec};
+use core::slice;
+
+// STACK INPUTS
+// ================================================================================================
+
+/// Initial state of the stack to support program execution.
+///
+/// The program execution expects the inputs to be a stack on the VM, and it will be stored in
+/// reversed order on this struct.
+#[derive(Clone, Debug, Default)]
+pub struct StackInputs {
```

Comment:
I would move stack inputs into the `core` crate as we need to use them also in the verifier.

Reply:
Done!